def test() {
    echo "Test Executed"
}

try {
    test
} catch(err) {
    echo "Error handled: ${err}"
}